src/components/Loader/Loader.module.css

```diff
@@ -0,0 +1,80 @@
+@keyframes LOADERPUFF {
+  0% {
+    transform: scale(0, 0);
+    opacity: 1;
+  }
+
+  100% {
+    transform: scale(0.9, 0.9);
+    opacity: 0;
+  }
+}
+
+.loader {
```

Comment:
With CSS modules, we usually follow a couple of slightly different naming conventions, since we don't need to worry about style collisions.
* The main class for the outermost DOM element wrapper of the component uses the same PascalCase name as the component itself, i.e. `.Loader`
* Child elements use camelCase class names, and don't need to be prefixed with the component name, i.e. `.icon` (although I also mentioned choosing a different name for this).
* Levels and sizes generally don't need to be prefixed (i.e. just `.sm`), although I'm not sure if this will scale for components that have lots of levels, sizes, states, etc.

Reply:
Super minor / non-blocker: I think `.loader` should still be `.Loader`